docsteady/tplan.py

```diff
@@ -139,6 +139,7 @@ def build_tpr_model(tplan_id):
         testresults, errors = TestResult().load(resp.json(), many=True)
         test_results_map[cycle_key] = {}
         for result in testresults:
+            result['sorted'] = sorted(result['script_results'], key=lambda step: step["index"])
```

Comment:
Are they actually unsorted? If so, there may be an issue in cycle.py, because they should be sorted as part of loading the schema:

https://github.com/lsst-dm/docsteady/blob/9d38736b2f5609a435b2a0c935550afa93e68e2f/docsteady/cycle.py#L152

Reply:
When the tests included in the test player are not executed, they are not sorted.
Maybe I should change in tplan.py from 
i: i["index"]
to 
step: step["index"]

Reply:
the problem can be seen in DMTR-161, commit 6fa6242.

Reply:
I made a few more tests and removing the second sort the problem is still visible in DMTR-161.
I have no idea what is wrong with the first sort, even if I use **step: step["index"]**.

@brianv0, if you can have a look today, we can try to fix the problem and get only one **sort**, otherwise, I suggest to keep the two sort command for the moment and open a Jira issue to solve it in the future.

Reply:
I think we should just keep both for now, we can try to check out the behavior a bit later